.github/workflows/functional-tests.yml

```diff
@@ -16,12 +16,12 @@ env:
 
 jobs:
   test:
-    runs-on: macos-latest
+    runs-on: macos-14
     timeout-minutes: 45
     strategy:
       matrix:
         jdk-version: ["17"]
-        ndk-version: ["21.4.7075529"]
+#        ndk-version: ["21.4.7075529"]
         api-level: [29]
```

Comment:
Not for this PR, but should be bump this to the latest? And also test on the minimum?

Reply:
Emulators beyond API 29 aren't supported by the GH action we use for running emulators yet